### ISSUE TITLE
Lesson pilot at Charité Universitäts Medizin Berlin

### ISSUE THREAD

**Issue by mhagdorn** [carpentries-incubator/python-intermediate-development#335]
Hi there,
I am planning to run the lessons on 5 morning sessions starting at the end of June, 2024. I will provide some feedback. 

Thanks a lot for putting together the course materials.

**Comment by anenadic** [comment]
Thank you for using the material. Please let us know the exact dates and numbers, once you know them. In instructor-led mode, we typically take 5 full days to deliver this, so the course does not feel rushed - just something for you to consider.

**Comment by anenadic** [comment]
Also note that we are [currently reworking Section 3](https://github.com/carpentries-incubator/python-intermediate-development/pull/333) (hopefully for the better) - it should be merged in time for your workshop.

**Comment by mhagdorn** [comment]
Thanks for the info, the dates are 24 June, 4, 5, 11 and 12 July. We are aiming for 20 participants but will let you know about the exact numbers. I think we will struggle to get people for 5 entire days. We shall see how it goes. We will certainly provide feedback.

**Comment by anenadic** [comment]
Hi @mhagdorn - the big changes to Section 3 have been merged yesterday - would be good for your trainers to have a quick look and see if they can spot any errors and inconsistencies and see how the material has changed.

**Comment by mhagdorn** [comment]
thanks for letting me know. I haven't really started the preparation yet but will need to do so soon. I'll let you know if I spot anything. Looking forward to teaching this course.

**Comment by anenadic** [comment]
Hi @mhagdorn - how did you pilot go? How many participants did you have in the end? Any feedback?

**Comment by mhagdorn** [comment]
Hi @anenadic,
sorry for not sending a message earlier. It dropped of my radar and I went on annual leave. I received a lot of feedback which was mostly very positive. The participants really enjoyed having the material which is quite excellent. We had a good mix of people with some quite experienced programmers. Everybody found something useful to them. We spread the course over 5 mornings which mostly worked ok, except for session 3. Next time around I will split that up into 2 sessions to give it enough space. The flow of the course was good. We had some really good group activities where we collected results on a flip chart. Next time around I will make more use of that. 

My organisation is using gitlab internally. So, it doesn't come as a surprise that people would have liked materials for gitlab. I would love to see that. I might be able to contribute something along those lines.

I have also found some minor issues. I will collate them and prepare a merge request.

Thanks again for providing the course material. We plan to run the course again next summer.
